src/graphql/admin/queries.ts

```diff
@@ -18,6 +19,7 @@ const QueryType = GT.Object({
     transactionsByHash: TransactionsByHashQuery,
     lightningInvoice: LightningInvoiceQuery,
     lightningPayment: LightningPaymentQuery,
+    wallets: WalletsQuery,
```

Comment:
We can make the query more specific, however my thinking was that it could be beneficial for an admin to have access to all the wallets and choose the fields they want. That would make the query more generic instead of specific to the use-case of proof of liabilities.

Reply:
ok fair. but in that case why not exposing the `accounts` first like in the main API? it's probably better if the API between `main` and `admin` have a similar pattern?